.github/workflows/run-prod-async.yml

```diff
@@ -0,0 +1,27 @@
+name: Upload all supported file types (prod) 
+
+on:
+  schedule:
+    - cron:  '0 0 * * * '
```

Comment:
This is midnight UTC, right? Which is 5pm PDT / 7pm CDT. I think that's a bit too close to when we expect users to be using our site to be adding additional load; let's do it closer to the middle of the night in US time zones.

This is also reminding me that we need to clean up test accounts and test data, since this would be about a 7x increase in how often we run the functional tests.